packages/contracts/src/networks.ts

```diff
@@ -302,7 +302,7 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     dripSize: '0.15',
     requiredEnvVariables: ['CHIADO_RPC_URL'],
     networkType: 'Testnet',
-    dripVersion: 0,
+    dripVersion: 1,
```

Comment:
This is intentional. I ran into some problems when testing gnosis due to the maximum drip limit. So I bumped this and redeployed the contracts to resolve the issue.